docs/sources/administration/roles-and-permissions/access-control/plan-rbac-rollout-strategy/index.md

```diff
@@ -301,3 +305,41 @@ roles:
 ```
 
 - Or use [RBAC HTTP API]({{< relref "../../../../developers/http_api/access_control/#update-a-role" >}}).
+
+### Manage user permissions through teams
+
+In the scenario where you want users to have granted access by the team they belong to, we recommend to set users role to `No Basic Role` and let the team assignment assign the role instead.
+
+1. In Grafana, ensure the following configuration settings are enabled.
+   1. `users.auto_assign_org = true` this will automatically assign users to the default organization.
+   1. `users.auto_assign_org_id = <org_id>` this will automatically assign users to the specified organization.
+   1. `users.auto_assign_org_role = None` this will automatically assign users to the specified role.
+   1. Restart the Grafana instance.
+2. Create a team with the desired name.
+3. Set the fixed roles needed for the team.
+4. Add users to the team.
+
+This way, a user will be added automatically to the default organization and have no permissions until it has been assigned to a team.
+
+### Allow user to access only to a specific dashboard
+
+In the scenario where you want to allow a user access to a specific dashboard, we recommend the `No Basic Role` as a starting point for the user and then assign the dashboard permissions to the user.
+However, in order to access the dashboard more permissions are needed other than just the dashboard. Here's a list of permissions needed in order to access a dashboard:
+
+1. The dashboard owner must give permission to the user
+1. The admin must grant the following permissions to the user
+   1. annotations:read
+   1. datasource:read
+
+The permissions list here are the bare minimum needed to be able to view the dashboard. However, keep in mind that if the dashboard has multiple features, the permissions needed to see the dashboard might be more than just the ones listed here.
+
+### Reducing scope of service accounts
+
+Service Accounts are great tools for M2M communications. However, they are also a security risk if not properly scoped. To reduce the scope of a service account, you can create a Service Account with `No Basic Role` as a starting point and then assign the permissions needed for the service account.
+
+1. Head over to the Service Accounts page and add a new Service Account.
+1. Set the necessary roles for the Service Account.
+   1. Set the basic role to `No Basic Role`.
```

Comment:
Same question here about how these substeps render. If they render as `1.`, then manually change these to a. and b.

Reply:
Same as before, I didn't like the final rendered style so I refactored it in order to keep some visual coherence.